examples/filter_5_string_bass_tabs.py

```diff
@@ -0,0 +1,28 @@
+import guitarpro
+import os
+
+def main(source):
+    print "Filtering...\n"
+    for dirpath, dirs, files in os.walk(source):
+    	for file in files:
+            try:
+                guitarProPath = os.path.join(dirpath, file)
+                tab = guitarpro.parse(guitarProPath)
+                for track in tab.tracks:
+                    if not track.isPercussionTrack:
+                        if len(track.strings) == 5:
+                            print guitarProPath
+            except:
+                pass
```

Comment:
Did you get any kind of exception here? Generally `except: pass` is an anti-pattern, so I need to know if it was caused by a bug in PyGuitarPro or something else.